test/type_construction_test.rb

```diff
@@ -9176,4 +9176,44 @@ def foo(x = 123, foo: true)
       end
     end
   end
+
+  def test_anonymouse_block_forwarding
```

Comment:
It looks like this test doesn't relate to anonymous block forwarding. Copy/paste error on the test method name?